include/deal.II/lac/sparse_matrix_ez.h

```diff
@@ -508,11 +508,9 @@ class SparseMatrixEZ : public Subscriptor
             const bool       col_indices_are_sorted = false);
 
   /**
-   * Copy the given matrix to this one.  The operation throws an error if the
-   * sparsity patterns of the two involved matrices do not point to the same
-   * object, since in this case the copy operation is cheaper. Since this
+   * Copy the given matrix to this one. Since this
    * operation is nonetheless not for free, we do not make it available through
-   * <tt>operator =</tt>, since this may lead to unwanted usage, e.g. in copy
+   * <tt>operator =</tt>, in which case may lead to unwanted usage, e.g. in copy
    * arguments to functions, which should really be arguments by reference.
```

Comment:
Good spot! The grammar of this sentence is completely unusable before as well as after the patch (so you didn't make it any worse :-)

How about writing this as follows:
```
Copy the matrix given as argument into the current object.

Since copying matrices is an expensive operation that we do not want to happen by accident through compiler generated code to <code>operator=</code> (as would happen, for example, if one accidentally declared a function argument of the current type <i>by value</i> rather than <i>by reference</i>), the functionality of copying matrices is implemented in this member function instead. All copy operations of objects of this type therefore require an explicit function call. 
```

If you feel ambitious, you can use similar text in the other matrix classes for `copy_from` as well :-)